pages/k8s/1.16/release-notes.md

```diff
@@ -0,0 +1,106 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Release notes"
+  description: Release notes for CDK
+keywords: kubernetes,  release, notes
+tags: [news]
+sidebar: k8smain-sidebar
+permalink: 1.16/release-notes.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+#### 1.16+ck2 - November 22, 2019 - [charmed-kubernetes-316](https://api.jujucharms.com/charmstore/v5/bundle/charmed-kubernetes-316/archive/bundle.yaml)
+
+#### 1.16+ck1 - October 4, 2019 - [charmed-kubernetes-270](https://api.jujucharms.com/charmstore/v5/bundle/charmed-kubernetes-270/archive/bundle.yaml)
+
+#### 1.16 - September 27, 2019 -  [charmed-kubernetes-252](https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-252/archive/bundle.yaml)
```

Comment:
The bug fix releases are missing the links to their respective milestones, which I think would be useful to include.